mautic/mautic
# New slot type for prefs center: Success Message

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | Yes
| Automated tests included? |
| Related user documentation PR URL | https://github.com/mautic/documentation/pull/280
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | https://github.com/mautic/mautic/issues/6053
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
Based on https://github.com/mautic/mautic/issues/6053 this PR added new slot type for Is preference center landing page.  

![image](https://user-images.githubusercontent.com/462477/39818473-47537c64-53a1-11e8-8810-462a21dc82a2.png)

#### Steps to test this PR:
1.  Enable preferences center in configurations and create landing page and enable Is preference center
2.  See new Success message slot type and add to your customized prefs center landing page
3. Create email and send emamil with customized prefs center landing page. Open unsubscribe link from email
4. See prefs center page, success message should be hidden
5. Save prefs center, success message should be visible

![image](https://user-images.githubusercontent.com/462477/39818673-c019c72a-53a1-11e8-9b61-9947c953c489.png)
